experimental/examples/falling-balls-mpp/build.gradle.kts

```diff
@@ -150,7 +151,22 @@ compose.desktop {
 
 compose.experimental {
     web.application {}
-    uikit.application {}
+    uikit.application {
+        bundleIdPrefix = "org.jetbrains"
+        projectName = "FallingBalls"
+        deployConfigurations {
+            simulator("IPhone8_en") {
```

Comment:
May be it'd be cleaner to have
```
simulator("IPhone8") {
    locale="en"
    ...
```
rather than encode the locale in the name.

Reply:
Yeah, sure! I will change

Reply:
Done.
Actually, language is not implemented yet.

Reply:
@alexander-gorshenev 
made your suggestion